etc/config-template.toml

```diff
@@ -710,3 +710,7 @@
 
 ## Stream channel window size, stream will be blocked on channel full.
 # stream-channel-window = 128
+
+[pessimistic-txn]
```

Comment:
I prefer
```
[transaction]
enable-pessimistic-txn = false
```

Reply:
TiDB's config is already `[pessimistic-txn]`.
We better keep it consistent.

Reply:
I think tikv can default enable it. Otherwise user will have to change both configurations.

Reply:
I do not agree with the first half-`default enable it` is not good. But we should consider using one configuration -- may be in the future